apps/Splash CLI/install

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+install_packages npm || exit 1
```

Comment:
You should install the Node.js app from pi-apps for `npm`.
```suggestion
"${DIRECTORY}/manage" install-if-not-installed Node.js || error "Node.js failed to install somehow!"```

Reply:
Does it contain NPM?

Reply:
> Does the Node.js app contain NPM?

Yes.